src/nvim/file_search.c

```diff
@@ -644,29 +644,9 @@ char_u *vim_findfile(void *search_ctx_arg)
               stackp->ffs_fix_path
               , stackp->ffs_wc_path
               ) == FAIL) {
-#ifdef FF_VERBOSE
```

Comment:
Given it already checks for `p_verbose >= 5`, maybe we could just remove the ifdefs? Not sure though, as I'm not sure how hot these regions are, so it might impact performance.